explore/src/main/scala/ColumnSelector.scala

```diff
@@ -14,10 +13,9 @@ import react.semanticui.modules.dropdown.*
 import reactST.{tanstackTableCore => raw}
 
 case class NewColumnSelector[T](
-  table:         raw.mod.Table[T],
-  columnNames:   Map[String, String],
-  hiddenColumns: View[List[TableColumnPref]],
-  clazz:         Css = Css.Empty
+  table:       raw.mod.Table[T],
+  columnNames: Map[String, String],
```

Comment:
is it a map of `name` to ...?

Reply:
id -> label

There was a `ColumnId` newtype which is lost here. I do think we should use it though but it wasn't straightforward to use it consistently everywhere. I'm thinking of adding it to the facade.

Reply:
I changed this to a function, it doesn't have to be a `Map`.